intellij_idea_set_up.md

```diff
@@ -0,0 +1,22 @@
+# IntelliJ IDEA settings
+
+### Installing the Bazel plugin
+Install it from within the IDE (`Settings > Plugins > Marketplace`, and search for **Bazel**).
+  
+  See another option to install the Bazel plugin - https://ij.bazel.build/docs/bazel-plugin.html#getting-started.
+  
+  At the time of writing the plugin is compatible with IntelliJ 2017.3.3 and later.
+
+### Importing a Project
+- Launch the IDE
+- Choose Import Bazel project from the Welcome screen or go to File - Import Bazel Project (if some project already opened in IDE)
+- First, you need to select an existing workspace root (the directory containing the WORKSPACE file).
+![Screenshot 1](https://imgur.com/WqwODKI.png)
+- Next, you need to choose `Create from scratch` option
+![Screenshot 2](https://imgur.com/WVMThTo.png)
+- Click Next then Finish to import the project into the IDE.
+
+See more import options - https://ij.bazel.build/docs/import-project.html
+
+### Building a Project
+Run `./compile.sh build` from IDE terminal if you get a mistake during building process in the previous step.
```

Comment:
How does that solve the mistake? Or is it a workaround?

Reply:
It is the workaround.

Reply:
### Building a Project
Run `./compile.sh build` from IDE terminal if you get a mistake during building process in the previous step.

### Troubleshooting
If the project doesn't build, you can try running `./compile.sh build` from IDE terminal.